src/globals/scss/__tests__/themes-test.js

```diff
@@ -72,10 +72,6 @@ const classic = [
   'accordion-title-margin',
   'accordion-content-padding',
 
-  // Card
-  'card-text-align',
```

Comment:
Is the removal here intentional...?

Reply:
@asudoh correct, it is intentional. Those `$card-` variables are no longer getting used anywhere. Maybe Carbon used to have a "card" component in the past?

This PR removes 4 variables, documented in the PR description.